### PR TITLE
[New Features] Disable Teleport Pad Messages & Bait List

### PR DESCRIPTION
**Pull request by Charzard4261**
Adds an option to disable "Teleported from x to y" and "This pad doesn't have a destination" messages. Off by default.

Adds a Bait List to the UI that shows bait images & count when holding a Fishing Rod. Easily expandable if new baits are added.

Edit: Having some issues with changing branch, trying to figure out what it's trying to do

I think it's fine?

### CODE REVIEW

File: src/main/java/codes/biscuit/skyblockaddons/listeners/PlayerListener.java [hunk 1]
```diff
@@ -326,6 +326,30 @@ public void onTick(TickEvent.ClientTickEvent e) {
...
+
+                if (!currentInventory.equals(previousInventory)) {
+                    BaitListManager.getInstance().refreshBaits();
+                }
+
+                previousInventory = currentInventory;
+            }
+
+            BaitListManager.getInstance().compareHeldItems(e.player.getHeldItem());
+        }
```
Comment (Berisan) [inline src/main/java/codes/biscuit/skyblockaddons/listeners/PlayerListener.java:381]: This looks like it belongs in `utils.InventoryUtils`. That way you can utilize the Item Pickup Log for watching bait changes.

Comment (Charzard4261) [reply src/main/java/codes/biscuit/skyblockaddons/listeners/PlayerListener.java:381]: I thought about doing that, but then if Feature.ITEM_PICKUP_LOG is disabled it looks like getInventoryDifference is never used. It also looks like it's specifically made for the log itself. Can you elaborate please?

Comment (Berisan) [reply src/main/java/codes/biscuit/skyblockaddons/listeners/PlayerListener.java:381]: You are right, right now it only checks for differences when the pickup log is enabled, as seen in [PlayerListener.java](https://github.com/BiscuitDevelopment/SkyblockAddons/blob/development/src/main/java/codes/biscuit/skyblockaddons/listeners/PlayerListener.java#L344). You could add the check for your feature there.

Indeed, it seems like in the long run the current logic of the pickup log should be split up into a difference checker and the Feature.ITEM_PICKUP_LOG for your feature to integrate in a simple way.

Comment (Charzard4261) [reply src/main/java/codes/biscuit/skyblockaddons/listeners/PlayerListener.java]: I modified the ItemPickupLog a bit, and properly implemented the PlayerTickEvent (Realised that it actually fires for other players too, but that should have been obvious haha). Could you check again please?

**Comment by Charzard4261** [comment]
@Berisan Done all of the changes you suggested, and thank you for finding that typo!

**Comment by Charzard4261** [comment]
Closing to move all changes into Charzard4261/SBA branch development, will make a new pull request when done. Git's giving me a hard time with resolving this conflict and this is the easiest way to get around it I think.

**Pull request closed without merging**